lib/reverse_string.rb

```diff
@@ -1,5 +1,26 @@
 def reverse_characters(str)
+    i = 0
+    text = str
+    arr_1 = []
+    while i < text.length
+        i += 1
+        arr_1 << text[-i]
+        new_string = arr_1.join
```

Comment:
So `new_string` is being re-defined every time you pop a character from the string off. Is that necessary? How would your method function differently if you were to only do `new_string = arr_1.join` **once**, after the `while` loop had completely in its entirety?